src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -406,119 +443,129 @@ public void run() {
+                                                    + " problem " + runSubmission.getProblem_id()
                                                     + " language " +  runSubmission.getLanguage_id()
                                                     + " entry_point " + runSubmission.getEntry_point()
-                                                    + " time " + overrideTimeMS 
+                                                    + " time " + overrideTimeMS
                                                     + " submissionID " + overrideSubmissionID);
                                                 try {
                                                     submitter.submitRun(runSubmission.getTeam_id(), runSubmission.getProblem_id(), runSubmission.getLanguage_id(),
                                                             runSubmission.getEntry_point(), mainFile, auxFiles, overrideTimeMS, overrideSubmissionID);
                                                 } catch (Exception e) {
-                                                    
+
                                                     // Send message, message will add to connectStatusTable
                                                     MessageManager.fireMessageListener(new MessageRecord("Unable to submit run " + overrideSubmissionID + " " + e.getMessage(), MessageScope.SHADOW_UI, e));
-                                                    
+
                                                     // TODO design error handling reporting
                                                     logAndDebugPrint(log, Level.WARNING, "Exception submitting run for event: " + event, e);
                                                 }
                                             }
-    
+
                                         } catch (Exception e) {
-                                            
+
                                             // Send message, message will add to connectStatusTable
                                             MessageManager.fireMessageListener(new MessageRecord("Exception processing event: " + event, MessageScope.SHADOW_UI, e));
-                                            
+
                                             // TODO design error handling reporting (logging?)
                                             logAndDebugPrint(log, Level.WARNING, "Exception processing event: " + event, e);
                                         }
-    
+
                                     } else if ("judgements".equals(eventType)) {
-                                        
+
+                                        if(!allowSubmitsAndJudgments) {
+                                             log.info("Skipping judgment event since the contest has not started and allowPrestartActivity is false.");
+                                            event = reader.readLine();
```

Comment:
why is that log.info at a different indent from the event line below it?

Reply:
Fixed in commit https://github.com/pc2ccs/pc2v9/pull/940/commits/fb5d45610571aba4c1e580e7e36bfe256ad6e3ef